buildSrc/src/main/groovy/org/gradle/build/docs/UserGuideSectionVerifierTask.groovy

```diff
@@ -0,0 +1,97 @@
+package org.gradle.build.docs
+
+import groovy.transform.CompileStatic
+import groovy.transform.ToString
+import org.gradle.api.DefaultTask
+import org.gradle.api.GradleException
+import org.gradle.api.file.FileCollection
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.TaskAction
+import org.w3c.dom.NodeList
+
+import javax.xml.parsers.DocumentBuilderFactory
+import javax.xml.xpath.XPathConstants
+import javax.xml.xpath.XPathFactory
+
+
+/**
+ * Created by pledbrook on 03/05/2016.
+ */
+@CompileStatic
+class UserGuideSectionVerifierTask extends DefaultTask {
```

Comment:
On the other hand, all the other tasks imported by _docs.gradle_ have a `Task` suffix :smile: